dbus.go

```diff
@@ -236,13 +221,72 @@ func (ss *SecretService) Delete(attrs map[string]string, expectedPassword string
 		return err
 	}
 
-	if promptPath != dbus.ObjectPath("/") {
-		fmt.Fprintln(os.Stderr, "TODO: Got prompt on delete. Sorry, can't do that yet")
+	if promptPath == dbus.ObjectPath("/") {
+		// no prompt. we're done now
+		return nil
+	}
+
+	ok, err := ss.prompt(promptPath)
```

Comment:
maybe wrap up ok&err into an error as you use it in the same way in a couple of locations.